src/services/http/httpService.js

```diff
@@ -28,6 +29,17 @@ export const httpService = {
         return response.json();
     },
 
+    getLocationsWithLatLon: async (lat, lon, allCheckboxes) => {
```

Comment:
I think this function should be modified.
Now, when no filters are checked, the accessability table shows no points:
<img src=https://github.com/user-attachments/assets/21a2de89-f22e-4a4a-bdd3-d81bb910f0b7 width=70% height=70%>
This is not consistent and not what we want. When no filters are checked, the closest points should be chosen from all the points.

Reply:
Another thing. When I have the table open and I check e.g. "pedestrians", the location "Zwierzyniecka" appears in the table. But when I uncheck it, the location still remains in the table:
<img src=https://github.com/user-attachments/assets/9e316878-9f82-4752-aedc-abccecf86f08 width=70% height=70%>

Reply:
Also - "Grunwaldzki" location should also be displayed in the scenario above. There is something not right with this location. When I have the table open and I try to check a filter, that would return **only** this "Grunwaldzki" location ("cars" or "big bridge") , an error occurs:
```
Uncaught Error: Objects are not valid as a React child (found: object with keys {displayValue, type, value}). If you meant to render a collection of children, use an array instead.
```